R/read_file.R

```diff
@@ -0,0 +1,80 @@
+#' Read files with extension .fst, .csv, and .rda (.rds and .RData)
+#'
+#' @param file_name Character; path to a file storing a single object.
+#'
+#' @noRd
+#' @examples
+#' # Odd case: .RData saved with extension .rda
+#' path <- tempfile(fileext = ".rda")
+#'
+#' data1 <- data.frame(x = 1)
+#' saveRDS(data1, path)
+#' read_file(path)
```

Comment:
Either the description or the concept behind this example/test is off. What is happening in the code here is that an RDS file is being saved with the file extension ".rda"